packages/gasket-plugin-swagger/package.json

```diff
@@ -38,13 +39,14 @@
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-plugin-swagger",
   "dependencies": {
     "@fastify/swagger": "^6.0.0",
+    "@types/express": "^4.17.21",
     "@types/swagger-ui-express": "^4.1.3",
-    "express": "^4.16.3",
     "swagger-jsdoc": "^4.0.0",
     "swagger-ui-express": "^4.1.4"
   },
   "devDependencies": {
     "@gasket/engine": "^6.43.0",
+    "@gasket/plugin-log": "^6.43.0",
```

Comment:
Added this because I changed the `express` and `fastify` hook tests to be more integration-y (using an actual Gasket plugin engine).